src/features/App/WorkspaceManager/ProfileCreator/index.tsx

```diff
@@ -130,6 +157,47 @@ export const ProfileCreator: FC<ProfileCreatorProps> = ({
 
 					{passwordError && <Text color="red.500">{passwordError}</Text>}
 				</VStack>
+
+				<HStack w="100%">
+					<Text
+						minW={'max-content'}
+						color="typography.additional"
+						fontSize="18px"
+					>
+						Encryption algorithm
+					</Text>
+					<Select
+						variant="secondary"
+						defaultValue="aes"
+						onChange={(evt) =>
+							setAlgorithm(evt.target.value as EncryptionAlgorithm)
+						}
+						disabled={isPending}
+					>
+						{[
+							{
+								value: ENCRYPTION_ALGORITHM.AES,
+								text: 'AES',
+							},
+							{
+								value: ENCRYPTION_ALGORITHM.TWOFISH,
+								text: 'Twofish',
+							},
+							{
+								value: ENCRYPTION_ALGORITHM.AES_TWOFISH,
+								text: 'AES - Twofish',
+							},
+							{
+								value: ENCRYPTION_ALGORITHM.TWOFISH_AES,
+								text: 'Twofish - AES',
+							},
```

Comment:
Let's introduce map where key is an algorithm code and value is an algorithm name, to ensure consistent naming.

Then we could do something like this

```tsx
[ENCRYPTION_ALGORITHM.AES, ENCRYPTION_ALGORITHM.TWOFISH, ...]
.map((algorithm) =>
  <option key={algorithm} value={algorithm}>
    {ENCRYPTION_ALGORITHM_NAMES[algorithm]}
  </option>
)
```